.circleci/config.yml

```diff
@@ -1,63 +1,165 @@
 version: 2
 jobs:
   build:
-    working_directory: ~/concur/developer.concur.com
+    working_directory: ~/test-repo
     docker:
-      - image: circleci/ruby:2.5.1
+      - image: cimg/ruby:2.7.3-node
+    environment:
+      BUNDLE_PATH: ~/repo/vendor/bundle 
     steps:
       - checkout
       - restore_cache:
           keys:
-          - devcenter-deps-{{ .Branch }}-{{ checksum "Gemfile" }}
-          - devcenter-deps-preview-      
-      - run: bundle install 
+            - rubygems-v1-{{ checksum "Gemfile.lock" }}
+            - rubygems-v1-fallback
+      - run: sudo apt-get install python3
```

Comment:
I'm not convinced we still need python.  Would be interesting to see if the build still works without installing python3, pip, pipend and Unipath

Reply:
You are right, since we don't use the python scripts anymore. Removed.